docs/testing.md

```diff
@@ -129,11 +129,24 @@ The audit will run against
 [Google's PageSpeed Insights](https://github.com/addyosmani/psi).
 
 
-# Django Server Unit Tests
+# Django and Python unit tests
 
-To run the server unit tests using Tox,
-make sure the `TOXENV` variable is set in your `.env` file and
-run `gulp test:unit:server` from the command-line in the project root.
+To run the the full suite of Python 2.7 unit tests using Tox, cd to the project 
+root, make sure the `TOXENV` variable is set in your `.env` file and then run
+
+```
+tox
+```
+
+If you haven't changed any installed packages and you don't need to test all migrations, you can run a much faster Python code test using:
+```
+tox -e fast
+```
+
+To see Python code coverage information, run
+```
+./show_coverage.sh
+``
```

Comment:
This whole expanded section of docs.

![good-job](https://cloud.githubusercontent.com/assets/10562538/19479310/0e96fd30-9514-11e6-95ae-33f3f56064e4.gif)